lib/nebulex_redis_adapter/redis_cluster/config_manager.ex

```diff
@@ -316,4 +317,7 @@ defmodule NebulexRedisAdapter.RedisCluster.ConfigManager do
   end
 
   # coveralls-ignore-stop
+
+  defp maybe_convert_to_integer(value) when is_binary(value), do: String.to_integer(value)
+  defp maybe_convert_to_integer(value), do: value
```

Comment:
LGTM, but let's move the `# coveralls-ignore-stop` above to skip these new lines (or you can also add a test, but I think it is just a simple thing).